tck/src/main/resources/internal/META-INF/microprofile-config.properties

```diff
@@ -138,6 +138,11 @@ tck.config.test.javaconfig.converter.implicit.sequence.ofBeforeValueOf=ofBeforeV
 tck.config.test.javaconfig.converter.implicit.sequence.valueOfBeforeParse=valueOfBeforeParse
 tck.config.test.javaconfig.converter.implicit.sequence.parseBeforeConstructor=parseBeforeConstructor
 
+# custom converters
+tck.config.test.javaconfig.custom.converter.integervalue=1
+tck.config.test.javaconfig.custom.converter.longvalue=1
+tck.config.test.javaconfig.custom.converter.doublevalue=1.0
+tck.config.test.javaconfig.custom.converter.booleanvalue=false
```

Comment:
@radcortez can you also add a charvalue here to ensure this value is not used.

Reply:
Done!